Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
+#[[ITK --start
```

Comment:
I was unaware of this block comment style until just now.

Reply:
I just found it as well.  It makes the code a lot cleaner for minimizing diffs.